index.js

```diff
@@ -92,26 +114,29 @@ const parseArgs = (
         // value and then increment pos so that we don't re-evaluate that
         // arg, else set value as undefined ie. --foo b --bar c, after setting
         // b as the value for foo, evaluate --bar next and skip 'b'
-        const val = options.withValue && options.withValue.includes(arg) ?
-          argv[++pos] :
+        const val = options.withValue &&
+          ArrayPrototypeIncludes(options.withValue, arg) ? argv[++pos] :
           undefined;
         // Append value to previous values array for case of multiples
         // option, else add to empty array
-        result.values[arg] = [].concat(
-          options.multiples && options.multiples.includes(arg) &&
+        result.values[arg] = ArrayPrototypeConcat(
+          [],
+          options.multiples &&
+            ArrayPrototypeIncludes(options.multiples, arg) &&
             result.values[arg] ?
-            result.values[arg] :
-            [],
-          val);
+            result.values[arg] : [],
+          val
+        );
       } else {
         // Cases when an arg is specified without a value, example
         // '--foo --bar' <- 'foo' and 'bar' flags should be set to true and
         // shave value as undefined
         result.flags[arg] = true;
         // Append undefined to previous values array for case of
         // multiples option, else add to empty array
-        result.values[arg] = [].concat(
-          options.multiples && options.multiples.includes(arg) &&
+        result.values[arg] = ArrayPrototypeConcat(
+          [],
+          options.multiples && ArrayPrototypeIncludes(options.multiples, arg) &&
             result.values[arg] ?
```

Comment:
`a ? a : b` should just be `a || b` (not part of this PR, but always a good fix to make)